redux/apis/api-types.ts

```diff
@@ -21,6 +21,9 @@ export interface SignUpRequest {
   local: boolean;
 }
 
+export interface SignInRequest
+  extends Omit<SignUpRequest, "confirmPassword" | "local"> {}
```

Comment:
is this interface empty?

Reply:
no, it is using the SignUpRequest but omitting `confirmPassword ` and `local`.